suite-native/module-accounts/src/components/AccountRenameModal.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+import { Box, BottomSheet, IconButton } from '@suite-native/atoms';
+
+import { AccountRenameForm } from './AccountRenameForm';
+
+type AccountRenameModalProps = {
+    accountKey: string;
+};
+
+export const AccountRenameModal = ({ accountKey }: AccountRenameModalProps) => {
```

Comment:
I would like to avoid naming things with `Modal` or `BottomSheet` suffix if there is more than just modal. For example here is Button that opens it. So better call it `AccountRenameButton`

Reply:
fixed in e44e7fee9